app/javascript/orangelight/multiselect_combobox.es6

```diff
@@ -130,15 +131,40 @@ export default class MultiselectCombobox {
   }
 
   #compare(a, b) {
+    const locations = [
```

Comment:
How are we going to keep this list in sync with other lists of libraries? It feels like one of those things we're going to forget about until it bites us.